defaults:
  - sliding_door_scenario
  - em@bopt_agent.gmm_generator: em_3_priors
  - override bopt@bopt_agent: bopt_gmm_force

bopt_agent:
  gmm_generator:
    type: em
    modalities: [position]
    model: <class 'bopt_gmm.gmm.instances.GMMCart3D'>
  debug_data_path: bopt_gmm_trajectories # Prefix for saving the successful trajectories used to train the GMM
  debug_gmm_path: bopt_gmm_model         # Prefix for saving the newly generated gmm
  base_accuracy: 0.65
  opt_dims:
    means: null
    cvars: null
